packages/extension/src/view/devtools/components/layout.tsx

```diff
@@ -0,0 +1,266 @@
+    isKeySelected,
+  } = useSidebar(({ state, actions }) => ({
+    activePanel: state.activePanel,
+    selectedItemKey: state.selectedItemKey,
+    currentItemKey: state.currentItemKey,
+    isSidebarFocused: state.isSidebarFocused,
+    updateSelectedItemKey: actions.updateSelectedItemKey,
+    isKeySelected: actions.isKeySelected,
+  }));
+
+  useEffect(() => {
+    setSidebarData((prev) => {
+      const data = { ...prev };
+      const psData = data['privacySandbox'];
+
+      psData.children['cookies'].panel = () => (
+        <Cookies setFilteredCookies={setFilteredCookies} />
+      );
+      psData.children['cookies'].children = Object.keys(tabFrames || {})
+        .filter((url) => {
+          if (url === ORPHANED_COOKIE_KEY) {
+            return frameHasCookies[url];
+          }
+
+          if (url === UNMAPPED_COOKIE_KEY) {
+            return frameHasCookies[url];
+          }
+
+          return true;
+        })
+        .reduce<SidebarItems>((acc, url) => {
+          let popupTitle = `Cookies used by frames from ${url}`;
+          let infoIconDescription = '';
+          if (url === ORPHANED_COOKIE_KEY) {
+            popupTitle = infoIconDescription =
+              'Frames that set these cookies were removed from the DOM, leaving these cookies orphaned.';
+          }
+
+          if (url === UNMAPPED_COOKIE_KEY) {
+            popupTitle = infoIconDescription =
+              'Cookies that could not be mapped to any frame.';
+          }
+
+          acc[url] = {
+            title: url,
+            popupTitle,
+            panel: () => <Cookies setFilteredCookies={setFilteredCookies} />,
+            icon: () => <CookieIcon />,
+            selectedIcon: () => <CookieIconWhite />,
+            infoIconDescription,
+            children: {},
+            isBlurred: !frameHasCookies?.[url],
+          };
+
+          return acc;
+        }, {});
+
+      const showInspectButton =
+        canStartInspecting && Boolean(Object.keys(tabFrames || {}).length);
+
+      if (showInspectButton) {
+        psData.children['cookies'].extraInterfaceToTitle = () => (
+          <InspectButton
+            isInspecting={isInspecting}
+            setIsInspecting={setIsInspecting}
+            isTabFocused={isSidebarFocused && isKeySelected('cookies')}
+          />
+        );
+      } else {
+        psData.children['cookies'].extraInterfaceToTitle = () => <></>;
+      }
+
+      return data;
+    });
+  }, [
+    canStartInspecting,
+    frameHasCookies,
+    isInspecting,
+    isKeySelected,
+    isSidebarFocused,
+    setIsInspecting,
+    setSidebarData,
+    tabFrames,
+  ]);
+
+  useEffect(() => {
+    if (Object.keys(tabFrames || {}).includes(currentItemKey || '')) {
+      setSelectedFrame(currentItemKey);
+    } else {
+      setSelectedFrame(null);
+    }
+  }, [currentItemKey, setSelectedFrame, tabFrames]);
+
+  useEffect(() => {
+    (async () => {
+      const tabId = chrome.devtools.inspectedWindow.tabId.toString();
```

Comment:
What is the advantage of using `chrome.devtools.inspectedWindow.tabId.toString();` over the utility function `getCurrentTabId`?